lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -177,12 +177,10 @@ swapNodes = (targetBody, existingNodes, options) ->
       else
         targetNode = targetNode.cloneNode(true)
         existingNode.parentNode.replaceChild(targetNode, existingNode)
-        if targetNode.nodeName == 'SCRIPT' && targetNode.getAttribute("data-turbolinks-eval") != "false"
-          executeScriptTag(targetNode)
   return
 
 executeScriptTags = ->
-  scripts = Array::slice.call document.body.querySelectorAll 'script:not([data-turbolinks-eval="false"])'
+  scripts = document.body.querySelectorAll 'script:not([data-turbolinks-eval="false"])'
```

Comment:
This was added in #189 because [`getElementsByTagName`](http://devdocs.io/dom/document.getelementsbytagname) returns a "live" node list. However, this now uses [`querySelectorAll`](http://devdocs.io/dom/document.queryselectorall), which doesn't return a live collection. See http://devdocs.io/dom/nodelist#Description

I added a regression test for this and verified that it doesn't pass when using `getElementsByTagName`.